libs/journeys/ui/src/components/Image/Image.stories.tsx

```diff
@@ -11,30 +11,29 @@ const Demo = {
   title: 'Journeys-Ui/Image'
 }
 
-const DefaultTemplate: Story<TreeBlock<ImageFields>> = ({ alt, ...props }) => (
-  <StoryCard>
-    <Image {...props} alt={alt} />
-  </StoryCard>
-)
-
-export const Default: Story<TreeBlock<ImageFields>> = DefaultTemplate.bind({})
-Default.args = {
+const imageProps = {
   id: 'Image',
   src: 'https://images.unsplash.com/photo-1508363778367-af363f107cbb?ixlib=rb-1.2.1&q=80&fm=jpg&crop=entropy&cs=tinysrgb&dl=chester-wade-hLP7lVm4KUE-unsplash.jpg&w=1920',
   alt: 'random image from unsplash',
   width: 1600,
   height: 1067,
   blurhash: 'L9AS}j^-0dVC4Tq[=~PATeXSV?aL'
 }
-Default.parameters = {
-  chromatic: { delay: 300 }
-}
 
-export const NoImageSource: Story<TreeBlock<ImageFields>> =
-  DefaultTemplate.bind({})
-NoImageSource.args = {
-  ...Default.args,
+const Template: Story<TreeBlock<ImageFields>> = ({ alt, ...args }) => (
+  <StoryCard>
+    <Image {...imageProps} {...args} alt={alt} />
```

Comment:
Why does the alt prop need to be seperated?

```suggestion
    <Image {...imageProps} {...args} />
```

Reply:
<img width="753" alt="image" src="https://user-images.githubusercontent.com/10802634/193939373-1c604e6f-e286-4cb3-888b-f3aad00766b1.png">

We get this error without it..